modules/react/loading-dots/lib/LoadingDots.tsx

```diff
@@ -1,73 +1,83 @@
 import * as React from 'react';
-import {keyframes} from '@emotion/react';
-import {canvas, borderRadius} from '@workday/canvas-kit-react/tokens';
-import {styled} from '@workday/canvas-kit-react/common';
+import {keyframes} from '@emotion/css';
+import {base, system} from '@workday/canvas-tokens-web';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStyles, handleCsProp, CSProps} from '@workday/canvas-kit-styling';
```

Comment:
Move the `keyframes` import to `@workday/canvas-kit-styling` so the static style transform understands it:
```suggestion
import {base, system} from '@workday/canvas-tokens-web';
import {createComponent} from '@workday/canvas-kit-react/common';
import {createStyles, handleCsProp, keyframes, CSProps} from '@workday/canvas-kit-styling';
```